src/lib/data/near.js

```diff
@@ -218,6 +218,7 @@ async function _initNear({
   selector,
   walletConnectCallback = () => {},
   customElements = {},
+  features = {},
```

Comment:
Let's add a comment above the `_initNear` that defines which features are available to keep track in one place. With description of a feature.